lib/Controller/RoomController.php

```diff
@@ -818,10 +818,11 @@ protected function formatParticipantList(array $participants, bool $includeStatu
 				}
 
 				if (isset($statuses[$userId])) {
+					$clearAt = $statuses[$userId]->getClearAt();
 					$result['status'] = $statuses[$userId]->getStatus();
 					$result['statusIcon'] = $statuses[$userId]->getIcon();
 					$result['statusMessage'] = $statuses[$userId]->getMessage();
-					$result['statusClearAt'] = $statuses[$userId]->getClearAt();
+					$result['statusClearAt'] = $clearAt ? $clearAt->getTimestamp() : null;
```

Comment:
Only suggestion:

If you like, you can change at other places of this PR but isn't requirement to merge this PR.

Since PHP 8.0 we can use nullsafe operator. This would be good to be used, is the same behavior when we to this at JS:

```javascript
err?.response?.data?.ocs?.data?.message
```

Ref: https://wiki.php.net/rfc/nullsafe_operator

```suggestion
					$result['statusClearAt'] = $statuses[$userId]->getClearAt()?->getTimestamp();
```

Reply:
yeah, but we backport this, so can't use it for now. But we can change in master afterwards